tests/xharness/Jenkins.cs

```diff
@@ -27,7 +27,7 @@ public class Jenkins
 		public bool IncludeiOSExtensions;
 		public bool ForceExtensionBuildOnly;
 		public bool IncludetvOS = true;
-		public bool IncludewatchOS = true;
+		public bool IncludewatchOS = false;
```

Comment:
you should start a meta-issue about things to review before a 11.4 based release

Reply:
Great idea!

Reply:
Created https://github.com/xamarin/xamarin-macios/issues/7818